apps/src/util/currentLocale.ts

```diff
@@ -0,0 +1,4 @@
+import {get} from 'js-cookie';
+import {DefaultLocale} from '@cdo/generated-scripts/sharedConstants';
+
+export default () => get('language_') || DefaultLocale;
```

Comment:
Does this mean we are not surfacing the locale anywhere else in the front end?
This will also allow tacking client-side events with language data.

Reply:
We do in the homepage/script data and `appOptions` via the `localesRedux` which this kind of replaces by looking at the `language_` cookie instead of the locale set within the rendered page. It might be nice to update the redux instead to use the default locale on its initial state (I think it is hard-coded there) and then update the redux to `setLocaleCode` to the cookie value.

The one complication that I'm not fully thinking through is that the locale on the ruby side is in the form `en-US` and on the JS side it is `en_us`. Not sure how much that matters in different situations.

Reply:
`window.appOptions` are set only on lab pages:

<img src="https://github.com/code-dot-org/code-dot-org/assets/11708250/4b6fabc4-a632-46ab-aac3-1b0548cbb5ad">

<img src="https://github.com/code-dot-org/code-dot-org/assets/11708250/6427bb2d-06f6-470e-a809-615eb76b4ffa">